src/stdlib_stats_distribution_PRNG.fypp

```diff
@@ -51,6 +50,8 @@ module stdlib_stats_distribution_PRNG
         integer :: k
 
         k = MAX_INT_BIT_SIZE - bit_size(n)
```

Comment:
Could the value `k` be made a parameter as follows:
```
integer, parameter :: k = MAX_INT_BIT_SIZE - bit_size(n) 
! or 
integer, parameter :: k = MAX_INT_BIT_SIZE - bit_size(1_${k1}$)
```

Reply:
Yes, both are correct here. I am not sure which one is better.

Reply:
I would go with the first then. It makes the intent clear.